shell/platform/windows/compositor.h

```diff
@@ -32,6 +35,9 @@ class Compositor {
 
   // Present Flutter content and platform views onto the view.
   virtual bool Present(const FlutterLayer** layers, size_t layers_count) = 0;
+
+ protected:
+  PlatformViewManager* platform_view_manager_;
```

Comment:
Consider documenting that this is a weak pointer, and that the PlatformViewManager is owned by FlutterEngine.

Reply:
Should we make this be a `std::shared_ptr` and have the Compositor store `std::weak_ptr`?